tests/e2e/run-scanner-v4.bats

```diff
@@ -2,10 +2,36 @@
 
 # Runs Scanner V4 tests using the Bats testing framework.
 
-setup_file() {
+init() {
     ROOT="$(cd "$(dirname "$BATS_TEST_FILENAME")"/../.. && pwd)"
     export ROOT
 
+    if [[ "${CI:-}" != "true" ]]; then
+        # Some friendly environment checks
+        if [[ -z "${BATS_CORE_ROOT:-}" ]]; then
+            echo "WARNING: You better set \$BATS_CORE_ROOT before executing this test suite." >&3
+            exit 1
+        fi
+    fi
+
+    # shellcheck source=../../scripts/ci/lib.sh
+    source "$ROOT/scripts/ci/lib.sh"
```

Comment:
Is it correct that those files sourced before each test case?

Reply:
Yes, because each test is executed in its own shell.